src/services/LiqualityService.ts

```diff
@@ -53,9 +55,15 @@ export default class LiqualityService extends WalletService {
 
   // eslint-disable-next-line class-methods-use-this
   async isConnected(): Promise<boolean> {
+    let continueCounting = true;
     return new Promise<boolean>((resolve, reject) => {
-      const walletAddresses: WalletAddress[] = [];
+      setTimeout(() => {
+        if (continueCounting) {
+          reject(new LiqualityError());
```

Comment:
I think you could resolve with false instead
```suggestion
          resolve(false);
```

Reply:
If the isConnected method resolves false, then we will try to reconnect, if the user disables the Liquality the thread is abandoned instead we´re throwing this exception.